website/docs/components/form/primitives/partials/code/component-api.md

```diff
@@ -34,6 +34,29 @@
   </C.Property>
 </Doc::ComponentApi>
 
+### Form::CharacterCount
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="controlId" @type="string">
+    The ID of the form control associated with the helper text. This is used to populate the `id` HTML attribute (with a `character-count-` prefix) of the element. This CharacterCount ID can then be referenced in the `aria-describedby` attribute of the form control.
+  </C.Property>
+  <C.Property @name="maxLength" @type="number">
+    The maximum number of characters allowed in the associated form element, used to determine the number of remaining characters. This does not restrict users from adding characters over the limit.
+  </C.Property>
+  <C.Property @name="minLength" @type="number">
+    The minimum number of characters required for the associated form element, used to determine the shortfall value.
+  </C.Property>
+  <C.Property @name="onInsert" @type="function">
+    Callback function invoked (if provided) when the associated input content is changed. The function receives as argument an object with the following properties: `inputControl` (a reference to the associated form control node element), `currentLength`, `maxLength`, `minLength`, `remaining`, and `shortfall`.
+  </C.Property>
+  <C.Property @name="yield">
+    Elements passed as children of this component are yielded inside the element. We only recommend using the block content for providing custom messages. The following variables are available within the block: `currentLength`, `maxLength`, `minLength`, `remaining`, and `shortfall`.
```

Comment:
It's not obvious what `remaining` and `shortfall` are/refer to, and how they're calculated.

Reply:
4b440ccd8